paddle/fluid/operators/box_coder_op.cc

```diff
@@ -70,7 +77,7 @@ class BoxCoderOpMaker : public framework::OpProtoAndCheckerMaker {
              "of variance.");
     AddInput(
         "TargetBox",
-        "(LoDTensor or Tensor) this input is a 2-D LoDTensor with shape "
+        "(LoDTensor or Tensor) this input is a 2-D/3-D LoDTensor with shape "
```

Comment:
This input can be a 2-D LoDTensor with shape [N, 4] when `code_type` is 'encode_center_size'. This input also can be a 3-D Tensor with shape [N, M, 4] when `code_type` is 'decode_center_size'.

Reply:
done